src/olympia/migrations/1052-remove-nullable-constraint-on-addon-contributions.sql

```diff
@@ -0,0 +1,5 @@
+-- Change NULL values to empty strings.
+UPDATE `addons` SET `contributions` = '' WHERE `contributions` IS NULL;
+
+-- Add NOT NULL constraint.
+ALTER TABLE `addons` MODIFY `contributions` VARCHAR(255) DEFAULT '' NOT NULL;
```

Comment:
I set `''` as a default value but it seems django would no set anything (default is `NULL`). Should I keep it or remove it?

Reply:
To keep it in line with Django, do not add `DEFAULT ''`. Django will store an empty string itself everytime you use the model, per its definition.

Reply:
Ok, I was not sure about that and could not find the info in the Django doc.